src/foundry/client/pixi/layers/effects/visibility.d.ts

```diff
@@ -0,0 +1,167 @@
+export {};
+
+declare global {
+  /**
+   * The visibility Layer which implements dynamic vision, lighting, and fog of war
+   * This layer uses an event-driven workflow to perform the minimal required calculation in response to changes.
+   * @see {@link PointSource}
+   */
+  class CanvasVisibility extends CanvasLayer {
+    /**
+     * The exploration container which tracks exploration progress
+     */
+    explored: PIXI.Container;
+
+    /**
+     * The container of current vision exploration
+     */
+    vision: CanvasVisionContainer;
+
+    /**
+     * The canonical line-of-sight polygon which defines current Token visibility.
+     */
+    los: PIXI.Graphics | undefined;
+
+    /**
+     * The optional visibility overlay sprite that should be drawn instead of the unexplored color in the fog of war.
+     */
+    visibilityOverlay: PIXI.Sprite;
+
+    /**
+     * The active vision source data object
+     * @defaultValue
+     * ```js
+     * {
+     *    source: undefined,
+     *    activeLightingOptions: {}
+     * }
+     * ```
+     */
+    visionModeData: { source: VisionSource | null; activeLightingOptions: Record<string, unknown> };
+
+    /**
+     * Define whether each lighting layer is enabled, required, or disabled by this vision mode.
+     * The value for each lighting channel is a number in LIGHTING_VISIBILITY
+     * ```js
+     * {
+     *   background: VisionMode.LIGHTING_VISIBILITY.ENABLED,
+     *    illumination: VisionMode.LIGHTING_VISIBILITY.ENABLED,
+     *    coloration: VisionMode.LIGHTING_VISIBILITY.ENABLED,
+     *    any: true
+     * }
+     * ```
+     */
+    lightingVisibility: {
+      illumination: CanvasVisibility.LightingVisibility;
+      background: CanvasVisibility.LightingVisibility;
+      coloration: CanvasVisibility.LightingVisibility;
+      any: boolean;
```

Comment:
Good to know that `any: boolean` haha. I guess it's even true from a TypeScript perspective. But `any: 1`, `any: "foo"`, `any: Document` etc. so.

(no action necessary, I'm just amused)